vortex-layout/src/layouts/flat/writer.rs

```diff
@@ -110,13 +113,14 @@ mod tests {
     use crate::segments::test::TestSegments;
     use crate::writer::LayoutWriterExt;
 
+    // Currently, flat layouts do not force compute stats during write, they only retain
```

Comment:
this was a test for retain_only but that no longer does anything in default setup

Reply:
I was thinking that if you ask the FlatLayoutWriter to include stats, it should probably compute missing ones and remove extra ones?

I didn't want to lump that into this change though since it could impact compression performance

Reply:
Ok, we can do it like that. I think it’s a nice feature overall